packages/graphql/src/schema-model/entity/model-adapters/ConcreteEntityAdapter.ts

```diff
@@ -80,9 +79,6 @@ export class ConcreteEntityAdapter {
 
             if (attributeAdapter.isConstrainable()) {
                 this.constrainableFieldsKeys.push(attribute.name);
-                if (attributeAdapter.isUnique()) {
```

Comment:
I believe the whole logic could `isConstrainable` can be removed from both here and Interfaces,  could you confirm this?